README.md

```diff
@@ -17,14 +17,30 @@ apt install libzstd-dev
 apt install cmake
 
 # default is release build
-mkdir build
-cd build
-cmake ..
-make
+./scripts/build.sh
+```
+
+If you are using an M1 mac, you will need to build this repository inside a linux docker container, as the build depends on
+some linux-specific libraries. You can run the following command to open a shell in a linux docker container with the current
+directory mounted:
```

Comment:
I would remove the first sentence, since this isn't specific to OS X. I use a similar `podman` command on RHEL, and more generally users should always be building & running BPF binaries inside a docker environment.